src/modules/launcher/PowerLauncher.UI/ResultList.xaml

```diff
@@ -57,7 +57,13 @@
                         <Image x:Name="AppIcon" Height="36" Margin="8,0,0,0" Grid.RowSpan="2" HorizontalAlignment="Left" Source="{Binding Image}" />
                         <TextBlock x:Name="Title" Grid.Column="1" Text="{Binding Result.Title}" FontWeight="SemiBold" FontSize="20" VerticalAlignment="Bottom"/>
                         <TextBlock x:Name="Path" Grid.Column="1" Text= "{Binding Result.SubTitle}" Grid.Row="1" Opacity="0.6" VerticalAlignment="Top"/>
-                        <ListView  Opacity="0" HorizontalAlignment="Right" Grid.RowSpan="2" Grid.Column="1" ItemsSource="{Binding ContextMenuItems}">
+                        <ListView  Opacity="0" 
+                                   HorizontalAlignment="Right" 
+                                   Grid.RowSpan="2" 
+                                   Grid.Column="1" 
+                                   ItemsSource="{Binding ContextMenuItems}" 
+                                   SelectionMode="Single"
+                                   SelectedIndex="{Binding SelectedIndex}">

```

Comment:
nit : SelectedIndex might be confusing with a similar variable name in resultsviewmodel. Should we rename it to something like `ContextMenuSelectedIndex ` ?

Reply:
> nit : SelectedIndex might be confusing with a similar variable name in resultsviewmodel. Should we rename it to something like `ContextMenuSelectedIndex ` ?

Thats a good call, I'll update